class/defaults.yml

```
@@ -1,6 +1,12 @@
 parameters:
   cert_exoscale:
-    =_metadata: {}
+    =_metadata:
+      deprecated: True
```

not sure if reclass-rs will parse this as a boolean! Probably better to use lower-case `true`


```suggestion
      deprecated: true
```